.infrastructure/60_cdn.tf

```diff
@@ -2,6 +2,15 @@ resource "aws_cloudfront_origin_access_identity" "main" {
   comment = "Identity to access S3 bucket."
 }
 
+## Function to manipulate the request
+resource "aws_cloudfront_function" "website_viewer_request_handler" {
+  name    = "website-viewer-request-handler"
+  runtime = "cloudfront-js-1.0"
+  # publish this version only if the env is true
+  publish = var.publish_cloudfront_functions
+  code    = file("${path.module}/../apps/cloudfront-functions/src/viewer-request-handler.js")
```

Comment:
I think the path is incorrect; it should be in `dist` instead of `src`

Reply:
correct